contracts/auth/MultisigBase.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IMultisigBase } from '../interfaces/IMultisigBase.sol';
+
+contract MultisigBase is IMultisigBase {
+    struct Voting {
+        uint256 voteCount;
+        mapping(address => bool) hasVoted;
+    }
+
+    struct Signers {
+        address[] accounts;
+        uint256 threshold;
```

Comment:
similar comment about not needing to store signer accounts across epochs, only the vote data

Reply:
As I've explained the reason in the previous GOV PR. You can't just override a struct in solidity storage. Or we can override the struct manually when rotating operators, but not sure if will save us any gas, because you need to loop and clear existing storage.

Reply:
ok, we can leave it as-is then